README.md

```diff
@@ -140,6 +140,36 @@ Available modules / packages:
 
 For more information about each component, check out our [Storybook](https://croz-ltd.github.io/tiller/?path=/docs/introduction--page).
 
+## Versions
+Every monday, a new version of the tiller packages will be published.
+Whether is minor or major will depend on the tasks completed the week before. If there is no new version on monday, it means there hasn't been enough changes to update the version.
+
+### Version ranges
```

Comment:
I think that this paragraph is a bit unnecessary as it only explains semantic versioning in npm. We should only mention that we stick to semantic versioning strictly and that breaking changes mean new major version